### PR TITLE
created models

### PR DESCRIPTION
**Pull request by phemmylintry**

### CODE REVIEW

File: app/models/models.py [hunk 1]
```diff
@@ -1,9 +1,69 @@
...
+
+
 class Test(db.Base):
     __tablename__ = "test"
     id = Column(String(255), primary_key=True, index=True, default=uuid4().hex)
     text = Column(String(255), index=True)
+
+
+class Village(db.Base):
+    __tablename__ = "villages"
```
Comment (nseetim) [inline app/models/models.py:17]: Nice Job
But I think we could just replace all the `uuid4().hex` with a defined constant just to keep it DRY

File: app/models/models.py [hunk 2]
```diff
@@ -1,9 +1,69 @@
...
+
+class Voter(db.Base):
+    __tablename__ = "voters"
+    id = Column(String(50), primary_key=True, index=True, default=uuid4().hex)
+    village = Column(String(255), ForeignKey("villages.id"))
+    name = Column(String(255), index=True)
+    contact = Column(String(255), ForeignKey("contacts.id"))
+    notes = Column(String(255), index=True)
+    importance = Column(String(255), index=True)
+    date_delivered = Column(DateTime, default=datetime.datetime.now)
```
Comment (nseetim) [inline app/models/models.py]: I believe the `default=datetime.datetime.now` should be `default=datetime.datetime.utcnow`

File: app/models/models.py (hunk 1 above)
Comment (phemmylintry) [reply app/models/models.py:17]: Do you mean to create a variable and use that?? @nseetim 

`
uuid = uuid4().hex
`

**Comment by nseetim** [comment]
Yeah
You could use something like UUID_HEX to avoid name conflict with uuid

**Pull request merged**